ironfish-cli/src/flags.ts

```diff
@@ -156,3 +156,14 @@ export const parseIron = (input: string, opts: IronOpts): Promise<bigint> => {
     }
   })
 }
+
+export const HexFlag = Flags.custom<string>({
+  parse: async (input, _ctx, opts) => {
+    const hexRegex = /^[0-9A-Fa-f]+$/g
+    if (!hexRegex.test(input)) {
+      throw new Error(`The hex string is invalid for field ${opts.name}`)
```

Comment:
Thinking about this error message a bit. I wonder if something like "The value provided for ${opts.name} is an invalid format. The value must only contain numbers and letters A-F" or something. I wouldn't want the expectation that a user should know or care what hex is.

Reply:
done